bot/exts/help_channels/_cog.py

```diff
@@ -14,14 +11,14 @@
 
 from bot import constants
 from bot.bot import Bot
+from bot.exts.help_channels import _channels
+from bot.exts.help_channels._names import MAX_CHANNELS_PER_CATEGORY, create_name_queue
```

Comment:
Feels inconsistent. Maybe better to just import `_names` as well and have consistent namespacing?

Reply:
This may have been changed in a later commit..

Reply:
It was still similar. Changed in 47d65fc

Reply:
nice, I like it. 👍🏼